projects/netgrif-components-core/src/lib/data-fields/enumeration-field/models/enumeration-field.ts

```diff
@@ -17,6 +19,7 @@ export enum EnumerationFieldValidation {
 }
 
 export class EnumerationField extends DataField<string> {
+    protected REQUEST_DEBOUNCE_TIME = 600;
```

Comment:
It would be better to have this value configurable. In this class, as a constructor argument with a default value.